connectors/salesforce/src/test/resources/connector/responses/successCardsWithCustomFields_xx.json

```diff
@@ -0,0 +1,320 @@
+{
+  "cards": [
+    {
+      "id": "00000000-0000-0000-0000-000000000000",
+      "name": "Salesforce",
+      "creation_date": "1970-01-01T00:00:00Z",
+      "template": {
+        "href": "https://hero/connectors/salesforce/templates/generic.hbs"
+      },
+      "header": {
+        "title": "Uber IT Account",
+        "subtitle": [
+          "Add activity"
+        ]
+      },
+      "body": {
+        "fields": [
+          {
+            "type": "GENERAL",
+            "title": "Account Owner",
```

Comment:
There are lots of things in here that aren't uppercase that should be coming from the _xx.properties.  See comment above ("Add activity" in the subtitle, for example is hard-coded in the java).

Reply:
Done. I have added the missed fields in ```text_xx.properties``` file and I have made the corresponding changes in ```successCardsWithCustomFields_xx.json```